jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -247,6 +258,16 @@
                for tenant in api.config.tenants.tenants
                if std.objectHas(tenant, 'mTLS')
              ] else []) +
+            (if std.objectHas(api.config.tenants, 'tenants') then [
+               if std.objectHas(tenant.oidc, 'configMapName') then {
+                 configMap: {
+                   name: tenant.oidc.configMapName,
+                 },
+                 name: tenant.name + '-tls-configmap',
+               }
+               for tenant in api.config.tenants.tenants
+               if std.objectHas(tenant, 'oidc')
```

Comment:
I probably need to add a second conditional here, like `if std.objectHas(tenant.oidc, 'caKey')`.